File: sk000801/입문/문자열 정렬하기(1).java
import java.util.*;
class Solution {
    public int[] solution(String my_string) {
        String[] numbers = my_string.replaceAll("[^0-9]","").split("");
        int[] answer = new int[numbers.length];
        for(int i = 0; i < answer.length; i++) {
            answer[i] = Integer.parseInt(numbers[i]);
        }
        Arrays.sort(answer);
        return answer;
    }
}

File: kyw0716/level0/문자열 정렬하기(1).js
function solution(my_string) {
  return my_string
    .replace(/[a-z]/g, "")
    .split("")
    .map((v) => parseInt(v, 10))
    .sort((a, b) => a - b);
}

// https://school.programmers.co.kr/learn/courses/30/lessons/120850
